code/game/gamemodes/cult/cult.dm

```diff
@@ -284,7 +284,7 @@ GLOBAL_LIST_EMPTY(all_cults)
 	if(!check_cult_victory())
 		feedback_set_details("round_end_result","cult win - cult win")
 		feedback_set("round_end_result",acolytes_survived)
-		to_chat(world, "<span class='danger'> <FONT size = 3> The cult wins! It has succeeded in serving its dark masters!</FONT></span>")
+		to_chat(world, "<span class='danger'> <FONT size = 3>The cult wins! It has succeeded in serving its dark masters!</FONT></span>")
```

Comment:
Can you also make these `<FONT` into `<font

Reply:
Donezo